app/controllers/users/passwords_controller.rb

```diff
@@ -21,13 +25,26 @@ def create
       redirect_to root_path and return
     end
 
-    super do |user|
-      if user.nil? || user.new_record?
-        flash.now[:notice] = ts("We couldn't find an account with that email address or username. Please try again?")
-      else
-        user.update_password_resets_requested
-        user.save
-      end
-    end
+    user.update_password_resets_requested
+    user.save
+
+    super
+  end
+
+  protected
+
+  # We need to include information about the user (the remaining reset attempts)
+  # in addition to the Archive's configured reset cooldown in the success
```

Comment:
Can we lowercase or remove "Archive" (e.g., Squidge) for the not-us folks who are using the code?